packages/cli/tests/e2e/collaboration/PermissionSPFX.tests.ts

```diff
@@ -28,6 +28,10 @@ describe("Collaboration", function () {
   const creator = process.env["M365_ACCOUNT_NAME"];
   let appId: string;
 
+  before(() => {
+    process.env.TEAMSFX_V3 = "true";
```

Comment:
Please use `mockedEnv` to mock the environment variables as blew:
![image](https://user-images.githubusercontent.com/15262146/226225520-4a2fb237-b802-4452-9545-cfaa83346627.png)

Reply:
Or remove this line because now it opens V3 by default.

Reply:
Updated, thx!